build.sbt

```diff
@@ -1,61 +1,92 @@
-name := "cl-snapshot-streaming"
+// scalafmt: { rewrite.scala3.convertToNewSyntax = false, rewrite.scala3.removeOptionalBraces = no }
+ThisBuild / organization := "org.constellation"
+ThisBuild / scalaVersion := "3.1.2"
+ThisBuild / version := "2.0.0-SNAPSHOT"
+ThisBuild / resolvers ++= List(Resolver.githubPackages("abankowski", "http-request-signer"))
+ThisBuild / scalafixDependencies += organizeImports
 
-version := "1.1.6"
-scalaVersion := "2.12.10"
-organization := "org.constellation"
+lazy val root = (project in file("."))
+  .settings(
+    name := "cl-snapshot-streaming",
+    testFrameworks += new TestFramework("weaver.framework.CatsEffect"),
+    libraryDependencies ++= catsDependencies ++ catsEffectDependencies ++ catsEffectTestDependencies ++ circeDependencies ++
+      configDependency ++ fs2Dependencies ++ http4sDependencies ++ loggingDepencencies ++ weaverDependency,
```

Comment:
I'd recommend the same pattern as we have in `tessellation` but that's up to you

Reply:
+1 for pattern from `tessellation`